perun-openapi/openapi.yml

```diff
@@ -8290,296 +8354,1032 @@ paths:
+          $ref: '#/components/responses/ExceptionResponse'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              title: InputUpdateServicesPackage
+              description: Input to updateServicesPackage
+              required:
+                - servicesPackage
+              properties:
+                servicesPackage:
+                  $ref: '#/components/schemas/ServicesPackage'
+
+  /urlinjsonout/servicesManager/addServiceToServicesPackage:
+    post:
+      tags:
+        - ServicesManager
+      operationId: addServiceToServicesPackage
+      summary: Adds a Service to a Services Package.
+      parameters:
+        - $ref: '#/components/parameters/servicesPackageId'
+        - $ref: '#/components/parameters/serviceId'
+      responses:
+        '200':
+          $ref: '#/components/responses/VoidResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+
+  /urlinjsonout/servicesManager/removeServiceFromServicesPackage:
+    post:
+      tags:
+        - ServicesManager
+      operationId: removeServiceFromServicesPackage
+      summary: Removes a Service from a Services Package.
+      parameters:
+        - $ref: '#/components/parameters/servicesPackageId'
+        - $ref: '#/components/parameters/serviceId'
+      responses:
+        '200':
+          $ref: '#/components/responses/VoidResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+
+  /json/servicesManager/getServicesFromServicesPackage:
+    get:
+      tags:
+        - ServicesManager
+      operationId: getServicesFromServicesPackage
+      summary: Lists services stored in a package.
+      parameters:
+        - $ref: '#/components/parameters/servicesPackageId'
+      responses:
+        '200':
+          $ref: '#/components/responses/ListOfServicesResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+
+  /urlinjsonout/servicesManager/addRequiredAttribute:
+    post:
+      tags:
+        - ServicesManager
+      operationId: addRequiredAttribute
+      summary: Mark the attribute as required for the service. Required attributes are requisite for Service to run.
+           If you add attribute which has a default attribute then this default attribute will be automatically add too.
+      parameters:
+        - $ref: '#/components/parameters/serviceId'
+        - $ref: '#/components/parameters/attributeId'
+      responses:
+        '200':
+          $ref: '#/components/responses/VoidResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+
+  /urlinjsonout/servicesManager/addRequiredAttributes:
+    post:
+      tags:
+        - ServicesManager
+      operationId: addRequiredAttributes
+      summary: Batch version of addRequiredAttribute.
+      parameters:
+        - $ref: '#/components/parameters/serviceId'
+        - name: attributes
```

Comment:
There is an already created schema named `attributes`. You can reuse it here. Moreover, when passing a list of values in a URL parameter, the name of the parameter should end with the `[]` brackets.